### PR TITLE
Misc. IMPORT updates

### LINKED ISSUE #5494: import: defaults are not accepted when importing
Examples of using import on our docs site look like this: 

```
IMPORT TABLE customers (
        id UUID PRIMARY KEY DEFAULT gen_random_uuid(),
        name TEXT,
        INDEX name_idx (name)
)
CSV DATA ('s3://acme-co/customers.csv?AWS_ACCESS_KEY_ID=[placeholder]&AWS_SECRET_ACCESS_KEY=[placeholder]&AWS_SESSION_TOKEN=[placeholder]')
;
```

However defaults are not accepted when importing. This will throw an error similar to this:
 
`pq: s3://acme-co/customers.csv?AWS_ACCESS_KEY_ID={PLACEHOLDER}&AWS_SECRET_ACCESS_KEY={PLACEHOLDER}: row 2: expected 2 fields, got 1`

We should either remove this example, or make a note that if you want to use a gen function with as a default, you're file should contain values for that column initially.

### LINKED ISSUE #5851: IMPORT Doc Update
Re: [IMPORT](https://cockroachlabs.com/docs//v19.2/import.html)

## Issue Description
document a CSV import flag `strict_quotes`. If import CSV file has quotes, multiple rows can be treated as single rows. strict_quotes flag will produce reasonable output to hint at an underlying issue ```row 1914: reading CSV record: parse error on line 1914, column 28: extraneous or missing " in quoted-field```
## Suggested Resolution 
document flag `strict_quotes` if import files have quotes within rows. This flag doesn't take any input. Example: ```CSV
        DATA ('gs://artem_cockroach_bucket/aka_name.csv')
WITH
    strict_quotes, delimiter = ',';```

### LINKED ISSUE #5895: import: add an example of DELIMITED with escaping
We ran into an input file like this today, with `|` characters as delimiters, and `\|` representing a literal `|`:

```
k1|v1
k2|v2
k3\|k3|v3
k4|v4\|\|\|v4
```

The proper way to import this is something like:
`import table kv (k string primary key, v string) delimited data ('nodelocal:///thing.psv') with fields_terminated_by='|', fields_escaped_by='\';`. It's possible to piece this together from the documentation of `delimited`, but we should at least include one example of how this works.

### PR DESCRIPTION
**Pull request by lnhsingh**
Edits:
- Add example of DELIMITED with escaping. Closes #5895.
- Add notes about DEFAULT values and how to set multiple options. Closes #5494, #5929.
- Add strict_quotes option. Closes #5851

### CODE REVIEW

**Comment by lnhsingh** [comment]
@dt - can you please review these misc. IMPORT updates? If you think someone else should review, lmk. Thanks!

File: v19.2/import.md [hunk 1]
```diff
@@ -78,14 +78,15 @@ URLs for the files you want to import must use the format shown below.  For exam
...
+You can control the `IMPORT` process's behavior using any of the following optional key-value pairs as a `kv_option`. To set multiple import options, use a comma separated list ([see examples](#import-a-delimited-data-file)).
 
 <a name="delimiter"></a>
 
 Key                 | Context&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Value                                                                                                                             |
 --------------------+-----------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 `delimiter`            | `CSV DATA `     | The unicode character that delimits columns in your rows. **Default: `,`**.
 `comment`              | `CSV DATA `     | The unicode character that identifies rows to skip.
+`strict_quotes`        | `CSV DATA`      | Use if CSV import files have quotes (`""`) within rows to prevent multiple rows from being treated as single rows. **Off by default**
```
Comment (Amruta-Ranade) [inline v19.2/import.md]: nit: Change "Off by default" to "Default: Off" to maintain consistency.

File: v20.1/import.md [hunk 2]
```diff
@@ -78,14 +78,15 @@ URLs for the files you want to import must use the format shown below.  For exam
...
+You can control the `IMPORT` process's behavior using any of the following optional key-value pairs as a `kv_option`. To set multiple import options, use a comma separated list ([see examples](#import-a-delimited-data-file)).
 
 <a name="delimiter"></a>
 
 Key                 | Context&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Value                                                                                                                             |
 --------------------+-----------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 `delimiter`            | `CSV DATA `     | The unicode character that delimits columns in your rows. **Default: `,`**.
 `comment`              | `CSV DATA `     | The unicode character that identifies rows to skip.
+`strict_quotes`        | `CSV DATA`      | Use if CSV import files have quotes (`""`) within rows to prevent multiple rows from being treated as single rows. **Off by default**
```
Comment (Amruta-Ranade) [inline v20.1/import.md]: nit: Change "Off by default" to "Default: Off" to maintain consistency.

File: v20.1/import.md [hunk 3]
```diff
@@ -78,14 +78,15 @@ URLs for the files you want to import must use the format shown below.  For exam
 
 ### Import options
 
-You can control the `IMPORT` process's behavior using any of the following optional key-value pairs as a `kv_option`.
+You can control the `IMPORT` process's behavior using any of the following optional key-value pairs as a `kv_option`. To set multiple import options, use a comma separated list ([see examples](#import-a-delimited-data-file)).
```
Comment (Amruta-Ranade) [inline v20.1/import.md]: nit: comma separated > comma-separated

File: v19.2/import.md [hunk 4]
```diff
@@ -78,14 +78,15 @@ URLs for the files you want to import must use the format shown below.  For exam
 
 ### Import options
 
-You can control the `IMPORT` process's behavior using any of the following optional key-value pairs as a `kv_option`.
+You can control the `IMPORT` process's behavior using any of the following optional key-value pairs as a `kv_option`. To set multiple import options, use a comma separated list ([see examples](#import-a-delimited-data-file)).
```
Comment (Amruta-Ranade) [inline v19.2/import.md]: nit: comma separated > comma-separated

**Pull request merged**